src/docs/cursor-ops.dox

```diff
@@ -161,4 +161,45 @@ that may not be modified or freed by the application.  If a longer scope
 is required, the application must make a copy of the memory before the
 cursor is re-used, closed or reset.
 
+@section cursor_bound Apply bounds to the cursor
+
+Cursors may have bounds applied to them. This sets a start and stop key for
```

Comment:
I would update this to be a bit clearer about what bounds are and why we need them.\
```
A cursor may have a set of bounds applied to it, this restricts the key range that the cursor operates within. 

Suppose we have a table with keys A to Z. We can restrict the cursor to only operate over keys L->Q. If the cursor walks past key Q, the cursor will enter an un-positioned state having walked off the "end" of the table from it's perspective. Additionally should the cursor perform a search_near it will only search within the keyrange of L->Q. This can provide a performance boost when operating within tables that have a large number of deleted or invisible records.

The cursor can have a lower bound set or an upper bound set, or both.

```

Reply:
[6b88eb1](https://github.com/wiredtiger/wiredtiger/pull/8367/commits/6b88eb16841dd50de96ea1436b961ec3a0e8644d)